src/judgements/Judgements.jl

```diff
@@ -6,7 +6,7 @@ module Judgements
     include("level.jl")
 
     export @newjudgement
-    export J, Judgement, NoJudgement, convert, rating, comment, certainty, judgement_key
+    export J, Judgement, NoJudgement, convert, rating, comment, certainty, judgement_key, keyword_func
```

Comment:
I dont think this should be exported, since this object only exists within the macro, the name of the function itself is determined by the newjudgement. Could be wrong though.

Reply:
It did not work without the export but I don't know.

Reply:
It still works without! Thank you.